src/renderer/scss/component/_header.scss

```diff
@@ -1,17 +1,37 @@
 .header {
+  position: fixed;
+  height: var(--header-height);
+  width: 100%;
   display: flex;
   z-index: 1;
   justify-content: space-between;
-  padding: $spacing-width $spacing-width 0 $spacing-width;
+  align-items: center;
+  padding: 0 $spacing-width;
   background-color: var(--color-bg);
+  box-shadow: var(--box-shadow-header);
+}
+
+.header__navigation {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__history {
+  display: flex;
+  padding: 0 $spacing-width * 1/2;
+
+  @media only screen and (min-width: $medium-breakpoint) {
```

Comment:
Is there a reason we set rules to only screen?

Reply:
I guess not since screen will be the only one used.

Reply:
`Media Types` are **optional** and if **not** used it defaults to `All`. Unless targeting a **printer** or **screen reader**, there is no need for them.